docs/INSTALL.md

```diff
@@ -0,0 +1,83 @@
+## Prerequisite
+If you are using kubernetes distribution with no running OLM you can use [operator-sdk] cli
```

Comment:
maybe s/kubernetes/kubernetes (except for OCP and OKD)

Reply:
ack